src/index.html

```diff
@@ -7,12 +7,71 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="products card"
+        class="card__img"
+      />
+
+      <h2 class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__text">Product code: 195434</p>
+      <div class="card__review-container">
+        <ul class="stars stars--4">
+          <li class="stars__star"></li>
+
+          <li class="stars__star"></li>
+
+          <li class="stars__star"></li>
+
+          <li class="stars__star"></li>
+
+          <li class="stars__star"></li>
+        </ul>
+
+        <p class="card__reveiews">Reviews: 5</p>
+      </div>
+
+      <p class="card__reviews"></p>
+
+      <div class="card__price-container">
+        <div class="card__price-label">Price:</div>
```

Comment:
You need to use the semantic tag for text, fix it in two places

Reply:
i used reference mate tags)) whats problems with them?)

Reply:
Not fixed.
Need to use semantic tag for text 
https://www.w3schools.com/html/html5_semantic_elements.asp
<img width="827" alt="image" src="https://github.com/user-attachments/assets/f64bb71a-f0bd-4d3f-98d7-2c4ef8cdda41">